lib/transforms/pull-request.js

```diff
@@ -79,4 +81,10 @@ module.exports = (payload, author) => {
       updateSequenceId: Date.now(),
     },
   };
+
+  if (pullRequestStatus === 'MERGED') {
```

Comment:
Could we add a comment here to indicate why this is the case to avoid someone changing it where they might not understand the consequences?